app/src/main/java/com/brandon/forzenbook/view/MainActivity.kt

```diff
@@ -29,8 +29,8 @@ class MainActivity : ComponentActivity() {
         super.onCreate(savedInstanceState)
         setContent {
             ForzenBookTheme {
-                val sizes = DpProvider()
-                CompositionLocalProvider(LocalSizeController provides sizes) {
+                val sizes = Dimens()
+                CompositionLocalProvider(LocalDimenController provides sizes) {
```

Comment:
`LocalDimens` would fit the composable naming conventions more

Reply:
renamed it to LocalDimens